src/main/java/com/bootcamp/demo/restapi/ScooterController.java

```diff
@@ -0,0 +1,42 @@
+package com.bootcamp.demo.restapi;
+
+import com.bootcamp.demo.model.Scooter;
+import com.bootcamp.demo.service.ScooterService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Set;
+
+@RestController
+@RequestMapping(path = "/api/scooters")
+public class ScooterController {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ScooterController.class);
+    private final ScooterService scooterService;
+
+    public ScooterController(ScooterService scooterService) {
+        this.scooterService = scooterService;
+    }
+
+    @GetMapping
+    public Set<Scooter> getScooters() {
+        LOGGER.info("GET SCOOTERS - API endpoint invoked");
+        return scooterService.findAllScooters();
+    }
+
+    @GetMapping("/{scooterId}")
+    public Scooter getScooterById(@PathVariable(value = "scooterId") String scooterId) {
+        LOGGER.info("GET SCOOTER BY ID - API endpoint invoked");
```

Comment:
As a general note for logging, it is usually best to try to include details which would help differentiate the logs from others. For example here, you could log the scooterId, in order to be able to see which call was for it (for example in case it would fail for one id, you could easily search afterwards for it and ease the identification of the cause).